node/as/http.js

```diff
@@ -335,9 +346,8 @@ AsHTTPHandler.prototype.handleRequest = function handleRequest(req, buildRespons
 
     function sendError(err) {
         if (!sent) {
-            // TODO: map type?
             sent = true;
-            buildResponse().sendError('UnexpectedError', err.message);
+            buildResponse().sendError('Declined', err.message);
```

Comment:
This should not be declined. If you want this semantics please `sendError(err, codeName)`